.github/workflows/dev-linux.yml

```diff
@@ -218,6 +218,35 @@ jobs:
           dirs: ${{ matrix.dirs }}
           handlers: ${{ matrix.handlers }}
 
+  ee_sqllogic_standalone:
+    name: sqllogic_standalone_${{ matrix.dirs }}_${{ matrix.format }}
+    runs-on: [ self-hosted, X64, Linux, 4c8g ]
+    if: github.repository == github.event.pull_request.head.repo.full_name
+    needs: build
+    strategy:
+      matrix:
+        dirs:
+          - "ee"
+        format:
+          - "parquet"
+        handlers:
+          - "mysql,http,clickhouse"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/test_ee_sqllogic_standalone_linux
+        timeout-minutes: 30
+        env:
+          DATABEND_ENTERPRISE_LICENSE: ${{ secrets.DATABEND_ENTERPRISE_LICENSE }}
```

Comment:
change to
QUERY_DATABEND_ENTERPRISE_LICENSE: ${{ secrets.DATABEND_ENTERPRISE_LICENSE }}

as databend-query shall load the env variable into its session scope settings